driver-mapping/src/main/java/com/datastax/driver/mapping/Mapper.java

```diff
@@ -116,7 +117,7 @@ public MappingManager getManager() {
 
     /**
      * Creates a query that can be used to save the provided entity.
-     * <p>
+     * <p/>
```

Comment:
We use unclosed `p` tags for paragraphs (mainly for historical reasons), this creates an empty paragraph so it's not the same.